frontend/public/locales/en/credentials.json

```diff
@@ -28,6 +28,8 @@
     "credentialsForm.submitButton.save": "Save",
     "credentialsForm.submitButton.saving": "Saving",
     "credentialsForm.cancelButton.label": "Cancel",
+    "credentialsForm.updated.title": "Credentials updated",
+    "credentialsForm.created.title": "Credentials created",
```

Comment:
Shouldn't be plural

Reply:
A secret might contain multiple credentials...

Reply:
We should get docs to review across the board the credential vs credentials.
But not in this issue.